ydb/public/lib/ydb_cli/commands/ydb_service_scheme.cpp

```diff
@@ -861,6 +926,43 @@ int TCommandDescribe::PrintTableResponsePretty(const NTable::TTableDescription&
     return EXIT_SUCCESS;
 }
 
+std::pair<TString, TString> TCommandDescribe::ParseTopicConsumer() const {
+    const size_t slashPos = Path.find_last_of('/');
+    std::pair<TString, TString> result;
+    if (slashPos != TString::npos && slashPos != Path.size() - 1) {
+        result.first = Path.substr(0, slashPos);
+        result.second = Path.substr(slashPos + 1);
+    }
+    return result;
+}
+
+int TCommandDescribe::TryTopicConsumerDescribeOrFail(TDriver& driver, const NScheme::TDescribePathResult& result) {
+    auto [topic, consumer] = ParseTopicConsumer();
+    if (!topic || !consumer) {
+        ThrowOnError(result); // no consumer can be found
+    }
+
+    NScheme::TSchemeClient client(driver);
+    NScheme::TDescribePathResult topicDescribeResult = client.DescribePath(
+        topic,
+        FillSettings(NScheme::TDescribePathSettings())
+    ).GetValueSync();
+    if (!topicDescribeResult.IsSuccess() || topicDescribeResult.GetEntry().Type != NScheme::ESchemeEntryType::Topic && topicDescribeResult.GetEntry().Type != NScheme::ESchemeEntryType::PqGroup) {
+        ThrowOnError(result); // return previous error, this is not topic
+    }
+
+    // OK, this is topic, check the consumer
+    NYdb::NTopic::TTopicClient topicClient(driver);
+    auto consumerDescription = topicClient.DescribeConsumer(topic, consumer, NYdb::NTopic::TDescribeConsumerSettings().IncludeStats(ShowPartitionStats)).GetValueSync();
+    ThrowOnError(consumerDescription);
+
+    return PrintDescription(this, OutputFormat, consumerDescription.GetConsumerDescription(), &TCommandDescribe::PrintConsumerResponsePretty);
```

Comment:
I think format will not be used here. Always pretty format.

Reply:
We use here the template function PrintDescription: https://github.com/ydb-platform/ydb/blob/main/ydb/public/lib/ydb_cli/commands/ydb_service_scheme.cpp#L323
In cheks for format enum value